exasol_integration_test_docker_environment/test/test_hash_temp_dir.py

```diff
@@ -206,12 +216,161 @@ def test_directory_file_names_not_equal_to_dir_names(self):
+                                    hash_file_names=True,
+                                    hash_directory_names=True,
+                                    hash_permissions=True,
+                                    use_relative_paths=True)
+        test_file1 = f"{self.test_dir1}/level0/test.txt"
+        with open(test_file1, "w") as f:
+            f.write("test")
+            hash1_content_only = hasher_content_only.hash([PathMapping("level0/test.txt", test_file1)])
+
+        test_file2 = f"{self.test_dir2}/level0/test.txt"
+        with open(test_file2, "w") as f:
+            f.write("test")
+            hash2_content_only = hasher_content_only.hash([PathMapping("level0/test.txt", test_file2)])
+
+        ascii_hash1_content_only = base64.b32encode(hash1_content_only).decode("ASCII")
+        ascii_hash2_content_only = base64.b32encode(hash2_content_only).decode("ASCII")
+        self.assertEqual(ascii_hash1_content_only, ascii_hash2_content_only)
+
+    def test_file_name_with_relative_path_in_different_sub_path(self):
+        """
+        Test that hashing of same files in different paths, and different subpaths, gives different result
+        """
+        hasher_content_only = \
+            FileDirectoryListHasher(followlinks=True,
+                                    hashfunc="sha256",
+                                    hash_file_names=True,
+                                    hash_directory_names=True,
+                                    hash_permissions=True,
+                                    use_relative_paths=True)
+        test_file1 = f"{self.test_dir1}/level0/test.txt"
+        with open(test_file1, "w") as f:
+            f.write("test")
+            hash1_content_only = hasher_content_only.hash([PathMapping("level0/test.txt", test_file1)])
+
+        test_file2 = f"{self.test_dir2}/level0/level1_0/test.txt"
+        with open(test_file2, "w") as f:
+            f.write("test")
+            hash2_content_only = hasher_content_only.hash([PathMapping("level0/level1_0/test.txt", test_file2)])
+
+        ascii_hash1_content_only = base64.b32encode(hash1_content_only).decode("ASCII")
+        ascii_hash2_content_only = base64.b32encode(hash2_content_only).decode("ASCII")
+        self.assertNotEqual(ascii_hash1_content_only, ascii_hash2_content_only)
+
+    def test_file_name_with_relative_path_in_relative_path_as_argument(self):
+        """
+        Test that hashing of same files in different paths, gives same result, using relative paths as argument
+        """
+        hasher_content_only = \
+            FileDirectoryListHasher(followlinks=True,
+                                    hashfunc="sha256",
+                                    hash_file_names=True,
+                                    hash_directory_names=True,
+                                    hash_permissions=True,
+                                    use_relative_paths=True)
+        test_file = f"test.txt"
```

Comment:
I think, you don't test what you are thinking

Reply:
Because , I think you are not using the newly crested files, instead you are using a file which happens to be created by setup_class

Reply:
As discussed by phone, this is correct.